### PR TITLE
Introduce template linter

### PR DESCRIPTION
**Pull request by rogatty**
## Links
* https://wikia-inc.atlassian.net/browse/XW-1234
* https://github.com/rwjblue/ember-cli-template-lint
* https://github.com/rwjblue/ember-template-lint/#rules

## Description

Introduce linting for our templates using ember-cli-template-lint. Enable only a single rule - `'block-indentation': 'tab'` - to keep PR readable while still providing value and base for introducing further rules.

Linter will be run on each `ember build` and `ember test` and display errors in console. It also means that our PR check "Front tests" will fail when rules are not followed.

## Reviewers

@Wikia/x-wing @dianafa @bkoval @kamilkoterba

### CODE REVIEW

File: front/main/app/templates/components/portable-infobox.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{{infoboxHTML}}}
 {{#if isLongInfobox}}
-<div class={{expandButtonClass}} {{action "toogleInfobox"}}>
-	<svg viewBox="0 0 12 7" class="icon pi-expand-icon"><use xlink:href="#chevron"></use></svg>
-</div>
+	<div class={{expandButtonClass}} {{action "toogleInfobox"}}>
+		<svg viewBox="0 0 12 7" class="icon pi-expand-icon"><use xlink:href="#chevron"></use></svg>
```
Comment (idradm) [inline front/main/app/templates/components/portable-infobox.hbs]: Shouldn't this be indented as well?

Comment (rogatty) [inline front/main/app/templates/components/portable-infobox.hbs]: Isn't it already? If you ask if we should put `<use>` in a seperate line then no, it seems like as long as everything is in a single line then this is fine. We can't do this, though:
```
<svg><use>
</use></svg>
```

File: front/main/ember-cli-build.js [hunk 2]
```diff
@@ -74,7 +74,6 @@ module.exports = function (defaults) {
 			// By default vendor is not watched by Ember CLI and we want to rebuild when common scripts are modified
 			vendor: 'vendor'
 		},
-		hinting: false,
```
Comment (bkoval) [inline front/main/ember-cli-build.js]: Just out of curiosity - what did that do? :)

**Comment by kvas-damian** [comment]
:+1:

File: front/main/ember-cli-build.js (hunk 2 above)
Comment (rogatty) [inline front/main/ember-cli-build.js]: Yeah, it was required before to disable JSHint. At some point it got obsolete and now is not needed. Also you can take a look at https://github.com/rwjblue/ember-cli-template-lint/issues/118

**Comment by hakubo** [comment]
LGTM

**Pull request merged**